### PR TITLE
Break endless loop in case of IOException.

### PR DESCRIPTION
**Pull request by matejonnet**

### CODE REVIEW

File: src/main/java/io/termd/core/http/HttpTtyConnection.java [hunk 1]
```diff
@@ -103,6 +102,16 @@ public String terminalType() {
 
   protected abstract void write(byte[] buffer);
 
+  /**
+   * Special case to handle tty events.
+   *
+   * @param bytes
+   */
+  public void writeToDecoder(byte[] bytes) {
+      lastAccessedTime = System.currentTimeMillis();
```
Comment (vietj) [inline src/main/java/io/termd/core/http/HttpTtyConnection.java]: correct indent size to 2 not 4

File: src/main/java/io/termd/core/http/HttpTtyConnection.java [hunk 2]
```diff
@@ -119,7 +128,7 @@ public void writeToDecoder(String msg) {
         case "read":
           lastAccessedTime = System.currentTimeMillis();
           String data = (String) obj.get("data");
-          decoder.write(data.getBytes()); //write back echo
+          decoder.write(data.getBytes());
```
Comment (vietj) [inline src/main/java/io/termd/core/http/HttpTtyConnection.java]: why did you remove the comment ?

File: src/main/java/io/termd/core/http/HttpTtyConnection.java (hunk 1 above)
Comment (matejonnet) [inline src/main/java/io/termd/core/http/HttpTtyConnection.java]: Damn ide settings.

File: src/main/java/io/termd/core/http/HttpTtyConnection.java (hunk 2 above)
Comment (matejonnet) [inline src/main/java/io/termd/core/http/HttpTtyConnection.java]: Mistake, reverting...
